frontend/src/components/common/Stack/Stack.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+interface IStackProps {
+  /** Defines the space between immediate children. */
+  gap?: string | number;
+  /** Defines the flex-direction style property. It is applied for all screen sizes. */
+  $direction?: 'row' | 'column';
```

Comment:
$?

Reply:
In styled-component speak if you prefix a property with "$" it will make sure to **not** passing it to the underlying DOM element. This is called a "transient prop". I had to use it here because **direction** is a valid DOM prop

https://styled-components.com/docs/api#transient-props

Reply:
OR we could choose a different name for that prop that doesn't clash with an existing DOM prop

Reply:
Thanks, I understand now.